enos/enos-scenario-upgrade.hcl

```diff
@@ -142,16 +145,16 @@ scenario "upgrade" {
     Bring the new upgraded binary from the artifactory to the instance running enos.
     EOF
```

Comment:
This description is no longer true as far as I can tell. It doesn't download the binary.